tests/e2e/scripts/keadm_e2e.sh

```diff
@@ -48,8 +48,26 @@ function build_image() {
   cd $KUBEEDGE_ROOT
   make image WHAT=cloudcore -f $KUBEEDGE_ROOT/Makefile
   make image WHAT=installation-package -f $KUBEEDGE_ROOT/Makefile
-  kind load docker-image kubeedge/cloudcore:$IMAGE_TAG --name test
-  kind load docker-image kubeedge/installation-package:$IMAGE_TAG --name test
+  # convert docker images to cri image, or cri runtime cannot identify the image that already existed on the local host
```

Comment:
I feel weird about this， if people use containerd, all the docker hub images can not be used directly?

Reply:
not know well, keadm join cannot locate image already existing in the local machine if we don't do this using `ctr` to load images

Reply:
I think we need to find out what's wrong

Reply:
ref: https://mxb.cc/posts/k8s-1.20-import-image-issue